src/HotChocolate/Core/HotChocolate.Core.sln

```diff
@@ -59,15 +59,27 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Validation.Tes
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "benchmark", "benchmark", "{A1F4B2A9-E0EA-4D8C-B76C-38F034C2E8DC}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HotChocolate.Validation.Benchmarks", "benchmark\Validation.Benchmarks\HotChocolate.Validation.Benchmarks.csproj", "{44517954-6231-4EE3-9F50-298C0EBFCDF5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Validation.Benchmarks", "benchmark\Validation.Benchmarks\HotChocolate.Validation.Benchmarks.csproj", "{44517954-6231-4EE3-9F50-298C0EBFCDF5}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HotChocolate.Utilities.Introspection", "..\Utilities\src\Utilities.Introspection\HotChocolate.Utilities.Introspection.csproj", "{39592692-847D-49F0-97DE-3F2F21677319}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Utilities.Introspection", "..\Utilities\src\Utilities.Introspection\HotChocolate.Utilities.Introspection.csproj", "{39592692-847D-49F0-97DE-3F2F21677319}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HotChocolate.Tests.Utilities", "test\Utilities\HotChocolate.Tests.Utilities.csproj", "{E2574A41-D9BF-4071-84D0-DB0F6D1148EE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Tests.Utilities", "test\Utilities\HotChocolate.Tests.Utilities.csproj", "{E2574A41-D9BF-4071-84D0-DB0F6D1148EE}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HotChocolate.Execution.Tests", "test\Execution.Tests\HotChocolate.Execution.Tests.csproj", "{1D3C84AE-2818-4873-A222-BE6D329E850D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Execution.Tests", "test\Execution.Tests\HotChocolate.Execution.Tests.csproj", "{1D3C84AE-2818-4873-A222-BE6D329E850D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HotChocolate.Execution.Benchmarks", "benchmark\Execution.Benchmarks\HotChocolate.Execution.Benchmarks.csproj", "{5B8D9924-2FE9-44EC-9584-D3ABCE87534B}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Execution.Benchmarks", "benchmark\Execution.Benchmarks\HotChocolate.Execution.Benchmarks.csproj", "{5B8D9924-2FE9-44EC-9584-D3ABCE87534B}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Types.Spatial", "src\Types.Spatial\HotChocolate.Types.Spatial.csproj", "{EE9747C9-7FED-4885-837C-C2E482BEA363}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Types.Tests.Spatial", "test\Types.Spatial.Tests\HotChocolate.Types.Tests.Spatial.csproj", "{9BD7D13A-AA46-4968-99EE-C2A80F333DE0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.AspNetCore", "..\AspNetCore\src\AspNetCore\HotChocolate.AspNetCore.csproj", "{7AFA44F6-5F42-4881-9884-F3B415032863}"

```

Comment:
These refrerences should really be removed after the playground is removed too!!